api/http/voters.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var Router = require('../../helpers/router');
+var httpApi = require('../../helpers/httpApi');
+
+/**
+ * Binds api with modules and creates common url.
+ * - End point: `/api/voters`
+ * - Public API:
+ * 	- get	/
+ * @memberof module:node
+ * @requires helpers/Router
+ * @requires helpers/httpApi
+ * @constructor
+ * @param {Object} votersModule - Module node instance.
+ * @param {scope} app - Network app.
+ */
+// Constructor
+function VotersHttpApi (votersModule, app) {
+
+	var router = new Router();
+
+	router.map(votersModule.shared, {
+		'get /': 'getVoters'
+	}, {responseWithCode: true});
+
+	httpApi.registerEndpoint('/api/voters', app, router, votersModule.isLoaded);
```

Comment:
Missed `isLoaded` function definition

Reply:
thanks!

Reply:
And yes, you are completely right. There is the middleware defined responsible for checking if module is loaded, so checking the module availability separately for every endpoint is redundant.